### ISSUE TITLE
Make an installer or package through pip/conda.

### ISSUE THREAD

**Issue by ChrisWellsWood** [woolfson-group/isambard#4]
The installation process is pretty complex at the moment, this needs to be addressed.

**Comment by ajasja** [comment]
Ooo, that would be nice:)

**Comment by ChrisWellsWood** [comment]
This has been addressed, you can now `pip install isambard`.

**Comment by ajasja** [comment]
I respectfully disagree. I had a horrible, horrible time installing this on windows (and I do know my way around:) A Conda package could ship with pre compiled pyx files:) (well or a binary wheel file would work as well)

**Comment by ChrisWellsWood** [comment]
It's a lot easier if you just follow the instructions and use Visual Studio and the pip installer.

**Comment by ajasja** [comment]
This is excellent:)

**Comment by ChrisWellsWood** [comment]
Now also `manylinux1` builds for 3.5/3.6.
